frontend/vitest.config.ts

```diff
@@ -66,8 +66,8 @@ export default defineConfig(() => {
     pool: 'threads',
     poolOptions: {
       threads: {
-        minThreads: threadCount,
-        maxThreads: threadCount,
+        minWorkers: 4,
+        maxWorkers: 4,
         useAtomics: true
       }
     }
```

Comment:
reading your comment in the ticket I think we can replace the `pool` + `poolOptions` altogether with:
```suggestion
    maxWorkers: 4
```
or `maxWorkers: threadCount`, although locally this seems to still create more processes than the maximum 😕 (and not limit the load of the entire test suite either).

Reply:
> locally this seems to still create more processes than the maximum 😕 (and not limit the load of the entire test suite either).

Yeah, same on CI, that's why I'm experimenting...